### ISSUE TITLE
which pokemon got which which number?

### ISSUE THREAD

**Issue by DarkriseOo** [mchristopher/PokemonGo-DesktopMap#44]

**Comment by fenixasin** [comment]
could you rephrase this question/issue?   if you mean which pokemon are what number please refer to http://bulbapedia.bulbagarden.net/wiki/Main_Page

**Comment by Trevor-van-L** [comment]
I'm going to close this issue, as it's not related to this project.

If there's actually substance here. please update with a description and re-open.

**Comment by DarkriseOo** [comment]
i mean, bevore starting the tool, i can choose which pokemon i want to see on map with put the right number for the pokemon in, but where do I know from which pokemon has which number?
